app/controllers/admins/base_controller.rb

```diff
@@ -1,3 +1,14 @@
 class Admins::BaseController < ApplicationController
+  before_action :is_admin?
```

Comment:
sửa lại tên nhé, return true/false mới để ?
```suggestion
  before_action :check_admin_role?
```